app-backend/tile/src/it/resources/application.conf

```diff
@@ -0,0 +1,18 @@
+gatling {
+  host = "https://app.staging.rasterfoundry.com"
+
+  refreshToken = ""
+  refreshToken = ${?REFRESH_TOKEN}
+
+  projectId = ""
+  projectId = ${?PROJECT_ID}
+
+  randomSeed = 42
+
+  tms {
+    template = "/tiles/${projectId}/$z/$x/$y?token=${authToken}"
```

Comment:
Sounds good. Does that mean we can forego the project reference as well a la cartodb or that "?token" should be something like "?mapToken"?